src/aleph_client/commands/files.py

```diff
@@ -132,3 +135,86 @@ async def download(
 
         logger.debug("File downloaded successfully.")
+
+
+@app.command()
+async def forget(
+    item_hash: str = typer.Argument(..., help="Hash to forget"),
+    reason: str = typer.Argument(..., help="reason to forget"),
+    channel: Optional[str] = typer.Option(None, help="channel"),
+    private_key: Optional[str] = typer.Option(
+        sdk_settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+    ),
+    private_key_file: Optional[Path] = typer.Option(
+        sdk_settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+    ),
+    debug: bool = False,
+):
+    """forget a file and his message on aleph.im."""
+
+    setup_logging(debug)
+
+    account: AccountFromPrivateKey = _load_account(private_key, private_key_file)
+
+    async with AuthenticatedAlephHttpClient(
+        account=account, api_server=sdk_settings.API_HOST
+    ) as client:
+        value = await client.forget(hashes=[item_hash], reason=reason, channel=channel)
+        typer.echo(f"{value[0].json(indent=4)}")
+
+
+class GetAccountFilesQueryParams(BaseModel):
+    pagination: int = Field(
+        default=100,
+        ge=0,
+        description="Maximum number of files to return. Specifying 0 removes this limit.",
+    )
+    page: int = Field(default=1, ge=1, description="Offset in pages. Starts at 1.")
+    sort_order: int = Field(
+        default=-1,
+        description="Order in which files should be listed: -1 means most recent messages first, 1 means older messages first.",
+    )
+
+
+# Your list command
+@app.command()
+def list(
+    address: Optional[str] = typer.Option(None, help="Address"),
+    private_key: Optional[str] = typer.Option(
+        sdk_settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+    ),
+    private_key_file: Optional[Path] = typer.Option(
+        sdk_settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+    ),
+    pagination: int = typer.Option(100, help="Maximum number of files to return."),
+    page: int = typer.Option(1, help="Offset in pages."),
+    sort_order: int = typer.Option(
+        -1,
+        help="Order in which files should be listed: -1 means most recent messages first, 1 means older messages first.",
+    ),
+):
+    account: AccountFromPrivateKey = _load_account(private_key, private_key_file)
+
+    if account and not address:
+        address = account.get_address()
+
+    if address:
+        # Build the query parameters
+        query_params = GetAccountFilesQueryParams(
+            pagination=pagination, page=page, sort_order=sort_order
+        )
+
+        uri = f"{sdk_settings.API_HOST}/api/v0/addresses/{address}/files"
+        with requests.get(uri, params=query_params.dict()) as response:
+            if response.status_code == 200:
+                balance_data = response.json()
+                formatted_balance_data = json.dumps(balance_data, indent=4)
+                typer.echo(formatted_balance_data)
```

Comment:
```suggestion
                files_data = response.json()
                formatted_files_data = json.dumps(files_data, indent=4)
                typer.echo(formatted_files_data)
```

Reply:
You also did a beautiful job on the `aleph node` commands. Think you could do the same for these?